client/packages/invoices/src/OutboundShipment/DetailView/OutboundLineEdit/hooks/useOutboundLineEditRows.ts

```diff
@@ -6,7 +6,8 @@ import { isA } from '../../../../utils';
 
 export const useOutboundLineEditRows = (
   rows: DraftOutboundLine[],
-  packSizeController: PackSizeController
+  packSizeController: PackSizeController,
+  batch?: string
```

Comment:
should this be scannedBatch vs just batch ?

Reply:
sure